management/src/main/java/com/bitwisekaizen/sdss/dto/IscsiTargetSpec.java

```diff
@@ -0,0 +1,49 @@
+package com.bitwisekaizen.sdss.dto;
+
+import java.util.List;
+
+/**
+ * Spec that describes the ISCSI target
+ */
+public class IscsiTargetSpec {
```

Comment:
Does it also changed when it's backed by Nexenta?  I mean, there's still going to be a UUID, IP, and the information about the target right?  Something we can iterate on but let's discuss before we productionize this.